### PR TITLE
configure app with postgres database

### PR DESCRIPTION
**Pull request by NyiingangoPacifique**
you need to have postgres installed and configured atleast having one user

Check readme for configuration

first pull this repo

- npm install

add .env file

add DB_URL variable and set value to it which containe your database credential

- DB_URL = postgres://<your_username>:<user_password>@<host>:<port>/<database>?sslmode=disable

after start server
- npm run dev

http://localhost:3000/api/user

### CODE REVIEW

File: src/models/user.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    unique: true
+},
+phone:{
+   type: Sequelize.STRING,
+   unique: true
+}
+},{
+   timestamps:false
+});
+
```
Comment (christianR15c) [inline src/models/user.js:19]: you should consider the model that allows association because we will need to use relations in tables

**Review by christianR15c — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+- `touch .env`
+
+- Enter line
+LOCAL_PORT = your port 
+- edit DB_URL basing on your database configuration
+DB_URL = postgres://<username>:<your_password>@localhost:5432/<your_database>?sslmode=disable
+
+- Go in `src/config/config.js` based on your database
+
+
```
Comment (David-Neza) [inline README.md:25]: create a proper folder structure that makes it easy to add dependencies for postgres and others.

File: src/models/user.js (hunk 1 above)
Comment (NyiingangoPacifique) [reply src/models/user.js:19]: on it

File: README.md (hunk 2 above)
Comment (NyiingangoPacifique) [reply README.md:25]: ok lets look at it

File: src/config/config.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+         min:0,
+         idle:10000
+   },
+   logging: false
+ });
+sequelize.authenticate().then(() => {
+     console.log('Connection has been established successfully.');
+}).catch(err => {
+     console.error('Unable to connect to the database:', err);
+});
```
Comment (Julesbi) [inline src/config/config.js:22]: Add a new line to look good

File: src/routes/user.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import "@babel/polyfill"
+import { createUser, getUser, getUsers, deleteUser, updateUsers } from '../controllers/user.controller';
+
+router.get('/', getUsers);
+router.get('/:id', getUser);
+router.post('/', createUser);
+router.delete('/:id', deleteUser);
+router.put('/:id', updateUsers);
+
+export default router;
```
Comment (Julesbi) [inline src/routes/user.js:12]: Add a new line to look good also.

File: src/models/user.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+export default User;
```
Comment (Julesbi) [inline src/models/user.js:20]: Add a new line to look good

**Review by Julesbi — CHANGES_REQUESTED** [review]
please work on all requested changes.

File: src/config/config.js (hunk 3 above)
Comment (NyiingangoPacifique) [reply src/config/config.js:22]: i don't understand what you are trying to say

Comment (Julesbi) [reply src/config/config.js:22]: I was saying to add a new line after the last line of code so that they will look good.

File: package.json [hunk 6]
```diff
@@ -21,21 +21,26 @@
...
   "homepage": "https://github.com/atlp-rwanda/Phantom-Backend-Pirates#readme",
   "dependencies": {
-    "@babel/node": "^7.16.8",
+    "@babel/polyfill": "^7.12.1",
     "@babel/runtime": "^7.17.2",
     "bcrypt": "^5.0.1",
+    "body-parser": "^1.19.2",
     "debug": "^4.3.3",
     "dotenv": "^16.0.0",
-    "express": "^4.17.2",
```
Comment (niyontwali) [inline package.json:29]: Why have two versions of express in your package.json file?

File: src/models/user.js (hunk 1 above)
Comment (niyontwali) [reply src/models/user.js:19]: You need to provide an explicit and detailed description of your PR. you can refer to this [link](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)

**Pull request closed without merging**